app.js

```diff
@@ -72,6 +78,10 @@ app.get('/deny', function(req, res) {
 	res.send("Denied request");
 });
 
+app.get('/add_directory', function(req, res) {
```

Comment:
I don't really care which as long we're consistent. If it's been dashes so far I'll swap it